_episodes/12-cluster.md

```diff
@@ -46,6 +47,14 @@ tasks.
 
 
 ## Logging in
+The first step in using a cluster is to establish a connection from our laptop to the cluster,
+via the Internet and/or your organisation's network. We use a program called the 
+Secure SHell (or ssh) client for this. (make sure you have a SSH client installed on your
+laptop. Refer the [setup]({{ page.root }}/setup) section for more details).
```

Comment:
Hey @willfurnass, when doing a github review or simply commenting on a PR, there is `+-` in the top bar of the comment window. With this, you can make direct suggestions, like the one below.

```suggestion
laptop. Refer to the [setup]({{ page.root }}/setup) section for more details).
```